lib/middleware.ts

```diff
@@ -86,6 +103,14 @@ var defaultHandlers = [
   (req, res) => ({"status-code": res.statusCode}),
   // Ip address
   (req) => ({ip: getIp(req)}),
+
+  // Kayvee's reserved fields
+  // Log level
+  (req, res) => ({level: getLogLevel(req, res)}),
+  // Source
+  () => ({source: "kayvee"}),
```

Comment:
Source should be a mandatory field, probably? Maybe the constructor should be `(source, clever_options, morgan_options)` instead of `(clever_options, morgan_options)`. It's a breaking change, but no one is using this yet anyway, so probably doesn't really matter